Dockerfile

```diff
@@ -201,7 +201,11 @@ COPY --from=charterafrica-builder --chown=nextjs:nodejs /workspace/node_modules
 
 # Since we can't use output: "standalone", copy all app's dependencies
 COPY --from=charterafrica-builder --chown=nextjs:nodejs /workspace/apps/charterafrica/node_modules ./apps/charterafrica/node_modules
-
+COPY --from=charterafrica-builder --chown=nextjs:nodejs /workspace/apps/charterafrica/next.config.js ./apps/charterafrica/next.config.js
+COPY --from=charterafrica-builder --chown=nextjs:nodejs /workspace/apps/charterafrica/.env ./apps/charterafrica/.env
+COPY --from=charterafrica-builder --chown=nextjs:nodejs /workspace/apps/charterafrica/migrations ./apps/charterafrica/migrations
```

Comment:
@koechkevin I still don't understand why we need the migrations. Isn't it only run once and not required after?

Reply:
@kelvinkipruto existing ones are not needed anymore but what if new migrations are added?
They'll need to be run on the server.